cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,71 @@
 /// <reference types="cypress" />
-/// <reference types="../support" />
+/// <reference types="../support" /
 
-describe('Settings page', () => {
-  before(() => {
+import { settings } from 'cluster';
+import SettingsPage from '../support/pages/settings.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
 
-  });
+const settingsPage = new SettingsPageObject();
```

Comment:
```suggestion
const settingsPage = new SettingsPageObject();
```
add this file to pull request